test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -1342,3 +1342,35 @@ foo``u{2195}abc
         }
     }
 }
+
+
+Describe 'Splatting' -Tags 'CI' {
```

Comment:
For a language feature, we need pretty intensive testing not just of the successful invocations but of error modes (such as parse errors) as well. For splatting in particular, given the complexity, I think the bar should be set high on testing.

Reply:
I agree but I'd like to defer this to the end when the implementation becomes final. Maybe it would be great to write down a couple of test cases as requirements that could be manually tested during development?